crates/rpc/rpc-types/src/eth/engine/payload.rs

```diff
@@ -10,6 +11,28 @@ use serde::{ser::SerializeMap, Deserialize, Serialize, Serializer};
 /// The execution payload body response that allows for `null` values.
 pub type ExecutionPayloadBodiesV1 = Vec<Option<ExecutionPayloadBodyV1>>;
 
+impl From<&Withdrawal> for StandaloneWithdraw {
+    fn from(withdrawal: &Withdrawal) -> Self {
+        StandaloneWithdraw {
+            index: withdrawal.index,
+            validator_index: withdrawal.validator_index,
+            address: withdrawal.address,
+            amount: withdrawal.amount,
+        }
+    }
+}
+
+impl From<StandaloneWithdraw> for Withdrawal {
+    fn from(standalone: StandaloneWithdraw) -> Self {
+        Withdrawal {
+            index: standalone.index,
+            validator_index: standalone.validator_index,
+            address: standalone.address,
+            amount: standalone.amount,
+        }
+    }
+}
```

Comment:
if we include the conversion here then we cannot get rid of it which defeats the point of moving the conversions

Reply:
My bad . Will fix

Reply:
If I implement the conversion in block.rs in primitives, then it leads to cyclic dependency error as I would have to import StandaloneWithdraw from rpc-types.
The. only way I can think is to duplicate block.rs as well?

Reply:
cc @mattsse

Reply:
right, we should convert this tryfrom into a function instead, like the other things we moved to compat